v1/chronos.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Chronos
+After=docker.service mesos-master.service
+Requires=docker.service mesos-master.service
+
+[Service]
+Environment=CHRONOS_IMAGE=mesosphere/chronos:chronos-2.3.3-1.0.1.ubuntu1404-mesos-0.22.0-1.0.ubuntu1404
+# NOTE: chronos doesn't need the `zk://`
+Environment=ZOOKEEPER=<ZK_ELB>
+
+Restart=on-failure
+RestartSec=20
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill chronos
+ExecStartPre=-/usr/bin/docker rm chronos
+ExecStartPre=/usr/bin/docker pull ${CHRONOS_IMAGE}
+ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
+    --name chronos \
+    --net=host \
```

Comment:
yes. these are based off of: https://github.com/mesosphere/coreos-setup/blob/master/etc/systemd/system/mesos-master.service#L16